Sites/GD2/Project 1/index.html

```diff
@@ -0,0 +1,71 @@
+<html lang="en">
```

Comment:
You're missing the starting line for your HTML doc. Add this on line 1 (don't delete what's currently there; what's on line 1 now should just get bumped down to line 2):
`<!DOCTYPE html>`